content/blog/nats-server-2.9.17-release.md

```diff
@@ -0,0 +1,124 @@
+
+This is possible by configuring the leafnodes as a local [cluster](https://docs.nats.io/running-a-nats-service/configuration/clustering) while each node also having the [`remotes`](https://docs.nats.io/running-a-nats-service/configuration/leafnodes/leafnode_conf#leafnode-remotes-entry-block) block configured establishing one or more connections to the remote cluster.
+
+As the scale of edge deployments increase, so does the need to optimize how real-time propagation of _subject-interest_. Where it was common to have many single leafnode deployments, having clusters of three or greater for edge deployment 3x-5x's the number of nodes in the fleet. For use cases such as electric vehicles and charging stations, the number of edge locations and leafnodes can be quite high!
+
+This release brings the first of a series of optimizations for increasingly large-scale fleets. This initial improvement accounts for the fact that interest-propagation updates only need to be sent to _one_ of the nodes in the leafnode cluster, rather than all of them. Once one node in the cluster receives these updates, it will then locally propagate these updates. This removes the propagation multiplier of running a leafnode cluster.
+
+**Relevant PRs**
+
+- [#4117](https://github.com/nats-io/nats-server/pull/4117)
+- [#4135](https://github.com/nats-io/nats-server/pull/4135)
+
+## ⏱️ Monotonic time calculations
+
+Although this improvement fixes a number of timing edge cases internally to the server, this section will serve as a Pro-Tip for anyone relying on Go's [time](https://pkg.go.dev/time) package for calculating durations using two time values.
+
+It is not uncommon for preferring UTC time values, especially in system spanning multiple time zones. However, did you know that calling the `time.Now().UTC()` will result in throwing away an internal monotonic value which differentiates two time values that are invoked within a nano-second level time frame, either in succession or concurrently. The [`time` package documentation calls this out](https://pkg.go.dev/time#hdr-Monotonic_Clocks).
+
+As a quick demonstration, the following code gets two time values in succession and then prints their values and the difference between them.
+
+```go
+package main
+
+import (
+	"fmt"
+	"time"
+)
+
+func main() {
+	t1 := time.Now()
+	t2 := time.Now()
+
+	fmt.Println("# Local (monotonic)")
+	fmt.Println(t1)
+	fmt.Println(t2)
+	fmt.Println(t2.Sub(t1))
+
+	fmt.Println("# UTC")
+	fmt.Println(t1.UTC())
+	fmt.Println(t2.UTC())
+	fmt.Println(t2.UTC().Sub(t1.UTC()))
+}
+```
+
+Below is an example output. Note the local times are differentiated by that `m=+` value shown in the output with a calculated 83ns difference between the times. The second set of times are converted to UTC before printing and performing the time substraction. Note how the times are no longer differentiated and the different is zero.
```

Comment:
The "with a calculated" is an additional comment, probably best to have a comma before it; however this one can be correct as-is if you mean that the actual delta is part of the differentiation. I'd insert the comma, were it me.

Near the end: "different" -> "difference" to get the noun.